views/index.erb

```diff
@@ -47,13 +110,21 @@
 
 <!-- Add your Javascript to this script tag -->
 <script language="javascript">
-  console.log("I am Javascript!");
 
-  var profile_values = document.getElementsByClassName('profile-value');
-  var total_words = 0;
+  function countWords(){
+  
+    var profile_values = document.getElementsByClassName("profile-value");
+  
+    var total_words = 0;
+  
+    for(var i = 0; i < profile_values.length; i++)
+    {
+        total_words += profile_values[i].innerHTML.split(' ').length;
```

Comment:
I see where you're going with this. Trouble is you might find that some of these values have whitespace in them, which might confuse split. Have a look at `profile_values[0].innerHTML.split(' ')` in your chrome console. Do you see any whitespace in there ? If so take a look at what `trim()` does and whether it can make your calculation more accurate.